client/src/components/ImageGallery/Swiper.scss

```diff
@@ -86,7 +86,7 @@
 
 @media screen and (max-width: 1220px){
     .c{
-        margin: 20px;
+        margin: 100px 20px 20px 20px;
```

Comment:
please follow best practices to keep it simple for reviewers,  we didnt comment on these syles,,  in order to avoid confusion and double work, you could just make notes for yourself, and fix them in a future individual hot fix branch if necessary

Reply:
that error occurred while I was merging gonna fox it sorry